tests/ssl-opt.sh

```diff
@@ -1946,6 +1946,43 @@ run_test    "Default, DTLS" \
             -s "Protocol is DTLSv1.2" \
             -s "Ciphersuite is TLS-ECDHE-RSA-WITH-CHACHA20-POLY1305-SHA256"
 
+requires_gnutls_tls1_3
+requires_config_enabled MBEDTLS_DEBUG_C
+requires_config_enabled MBEDTLS_SSL_SRV_C
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
```

Comment:
Don't we require both TLS 1.2 and TLS 1.3 for all of these tests, since they test version negotiation in hybrid mode?

Reply:
The test is valuable also in the case where the server supports only TLS 1.2: case of a TLS 1.2 and TLS 1.3 capable client trying to connect to the Mbed TLS TLS 1.2 only server. But this case is not well aligned with the test description. I've tried to improve that.